### PR TITLE
Save for later tones, and meta fixes

### PR DESCRIPTION
**Pull request by zeftilldeath**
Yay more deletions than additions.

# Byline image
When there's a byline image the date stamp wasn't playing nicely with the float, and was stacking beneath despite there being plenty of room.

Before
screen shot 2016-11-23 at 16 33 52

After
screen shot 2016-11-23 at 16 34 01

# Byline image
The expanded state of the sharing buttons was being clipped by an overflow:hidden coming from a clearfix script, updated it to a newer one that doesn't rely on overflow. 

Before
screen shot 2016-11-23 at 15 37 20

After
screen shot 2016-11-23 at 15 36 26

# Content age notice
Moved it outside of the sharing buttons div, resulting in no jump when you open the sharing options.

# Save for later
Save for later appears in a different tone colour all over the shop, considering the button performs the same action, it may as well remain consistent, unless it's a media tone of course. Furthermore the hover state of saved had a disappearing icon.

screen shot 2016-11-23 at 15 36 10

Now it is consistently blue, as a lot of our link language is. 
screen shot 2016-11-23 at 16 41 23

Have a gander please @gtrufitt 👁 👁

### CODE REVIEW

**Comment by zeftilldeath** [comment]
Please take note, these are the most coherent commits of my whole life. @OliverJAsh would have been proud.

File: static/src/stylesheets/module/content/_content.scss [hunk 1]
```diff
@@ -1115,9 +1112,8 @@
         }
 
         @include mq($until: leftCol) {
-            float: left;
             position: relative;
-            margin-top: -$gs-baseline/4;
+            margin-top: -4px;
```
Comment (sndrs) [inline static/src/stylesheets/module/content/_content.scss]: `4`?

Comment (gtrufitt) [inline static/src/stylesheets/module/content/_content.scss]: magic

File: static/src/stylesheets/module/content/_content.scss [hunk 2]
```diff
@@ -587,8 +586,7 @@
 .byline {
     @include fs-bodyHeading(2);
     margin-bottom: 0;
-    float: left;
-    margin-right: $gs-gutter/4;
+    display: inline;
```
Comment (gtrufitt) [inline static/src/stylesheets/module/content/_content.scss]: Sure you don't want `inline-block` here?

File: static/src/stylesheets/module/_save-for-later.scss [hunk 3]
```diff
@@ -1,101 +1,76 @@
...
+        border: 1px solid $news-main-1;
+        fill: $news-main-1;
 
         svg {
             width: 8px;
         }
     }
 }
 
-// colours per state
```
Comment (gtrufitt) [inline static/src/stylesheets/module/_save-for-later.scss]: gr8!

File: static/src/stylesheets/module/_save-for-later.scss [hunk 4]
```diff
@@ -1,101 +1,76 @@
...
-@include sfl-button-toned(dead, $live-default, $neutral-7);
-@include sfl-button-toned(editorial, $editorial-default);
-@include sfl-button-toned(feature, $feature-default);
-@include sfl-button-toned(letters, $comment-default);
-@include sfl-button-toned(live, $live-default, $neutral-8);
-@include sfl-button-toned(media, $media-default, $media-background);
-@include sfl-button-toned(news);
-@include sfl-button-toned(review, $review-background);
+.save-for-later__button--saved,
+.save-for-later__button--save:hover {
```
Comment (gtrufitt) [inline static/src/stylesheets/module/_save-for-later.scss]: Can you add `:focus` too while you're here plz thx

**Review by gtrufitt — APPROVED** [review]

**Comment by zeftilldeath** [comment]
Acted on all of the feedback, thanks @gtrufitt and @sndrs !

**Pull request merged**